dev/manual_tests/overlay_geometry.dart

```diff
@@ -163,8 +163,9 @@ class OverlayGeometryAppState extends State<OverlayGeometryApp> {
     });
   }
 
-  void handleTapUp(GlobalKey target, Point globalPosition) {
+  void handleTapUp(Key t, Point globalPosition) {
     setState(() {
+      GlobalKey target = t;
       markers[MarkerType.touch] = globalPosition;
```

Comment:
Please don't make this change. If we only support GlobalKeys in this function, then the function should take a GlobalKey, so that the API is clearly documented.